pandas/tests/scalar/test_timedelta.py

```diff
@@ -105,15 +105,6 @@ def test_timedelta_ops_scalar(self):
             result = base - offset
             assert result == expected_sub
```

Comment:
move this to the sub module as well

Reply:
This will get moved (and parametrized) in an upcoming pass.  For right now this PR is intended to be a minimal set that will allow for #19378 to go through.

Reply:
better to move it now

Reply:
OK, but just this one...